Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml

```
@@ -16,6 +16,13 @@
     - !type:DepartmentTimeRequirement
       department: Command
       time: 36000 # 10 hours
+    - !type:AgeRequirement
+      requiredAge: 21
+    # Goobstation - Siouxcide, prevent felinids from playing command
+    - !type:SpeciesRequirement
+      inverted: true
+      species:
+      - Felinid
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)